helm/content-store/templates/configmap.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-nginx-conf
+data:
+  nginx.conf: |-
+    user  nginx;
+    worker_processes  1;
```

Comment:
This is already the default and we're not likely to need to change it, so I'd probably omit it.
```suggestion
```